.github/workflows/test.yml

```diff
@@ -23,17 +43,27 @@ jobs:
           - "5.22"
           - "5.20"
 
-    container:
-      image: perldocker/perl-tester:${{ matrix.perl-version }}
-
+    name: Perl ${{ matrix.perl }} on ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v2
 
+      - name: Set up perl
+        uses: shogo82148/actions-setup-perl@v1
+        with:
+          perl-version: ${{ matrix.perl }}
+
+      - uses: actions/download-artifact@master
```

Comment:
You might want to use a version here:

`actions/download-artifact@v2`

In particular, you can avoid warnings if/when the main branch name changes.